src/test/java/seedu/address/logic/parser/classcommands/FindClassCommandParserTest.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.logic.parser.classcommands;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.classcommands.FindClassCommand;
+import seedu.address.logic.parser.classcommandparsers.FindClassCommandParser;
+import seedu.address.model.tuitionclass.predicates.ClassTimingContainsKeywordsPredicate;
+
+public class FindClassCommandParserTest {
+
+    private FindClassCommandParser parser = new FindClassCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        assertParseFailure(parser, "     ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindClassCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validDayArgs_returnsFindClassCommand() {
+        FindClassCommand expectedFindClassCommand =
+                new FindClassCommand(new ClassTimingContainsKeywordsPredicate(Arrays.asList("FRI")));
+        assertParseSuccess(parser, "FRI", expectedFindClassCommand);
+    }
+
+    @Test
+    public void parse_invalidDayArgs_throwsParseException() {
+        assertParseFailure(parser, "FRO",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindClassCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_invalidMultiDayArgs_throwsParseException() {
+        assertParseFailure(parser, "FRI SAN",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindClassCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validMultiDayArgs_returnsFindClassCommand() {
+        FindClassCommand expectedFindClassCommand =
+                new FindClassCommand(new ClassTimingContainsKeywordsPredicate(Arrays.asList("FRI", "SAT")));
+        assertParseSuccess(parser, "FRI SAT", expectedFindClassCommand);
```

Comment:
If this parser accepts multiple keywords, how come it doesnt find the classes with fri and sat for the command? The behaviour is not consistent with the findname command

Reply:
Other than this, LGTM

Reply:
I guess we have to make it such that it only accepts `FRI 0800-1000` in the UG